ambari/datadog_checks/ambari/ambari.py

```diff
@@ -2,16 +2,14 @@
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
 from datadog_checks.base import AgentCheck
-from six.moves.urllib.parse import quote, urljoin
```

Comment:
Not used, if you use an IDE like pycharm it will show unused imports and other style issues in grey or yellow